File: Codewars/8_kyu/Sum_of_positive.js
// You get an array of numbers, return the sum of all of the positives ones.

// Example [1,-4,7,12] => 1 + 7 + 12 = 20

// Note: if there is nothing to sum, the sum is default to 0.

// My Solution
function positiveSum(arr) {
    let res = 0
    arr.forEach((a) => {
        if (a >= 0) {
            res += a
        }
    })
    return res
}

File: Codewars/Go/8_kyu/Sum of positive.go
// Description:
// You get an array of numbers, return the sum of all of the positives ones.

// Example [1,-4,7,12] => 1 + 7 + 12 = 20

// Note: if there is nothing to sum, the sum is default to 0.

// My Solution:
package kata

func PositiveSum(numbers []int) int {
  res := 0
  for _,v := range numbers{
    if v > 0 {
      res += v
    }
  }
  return res
}


// Another way:
package kata

func PositiveSum(numbers []int) (sum int) {
  for _, num := range numbers {
    if num > 0 {
      sum = sum + num
    }
  }
  return
}